src/scalib/postprocessing/leakage_to_success.py

```diff
@@ -0,0 +1,331 @@
+r"""Lower bounds on the log-guessing entropy, log of the guessing entropy, log of the median rank and upper bound on the proability of a successful attack in presence of key enumeration.
```

Comment:
It would be nice to explain add explanations for novice users who do not really know what they need. E.g., intuition of what those functions are, typical case where they would be used.